starknet_devnet/blueprints/rpc/call.py

```diff
@@ -36,6 +43,8 @@ async def call(request: FunctionCall, block_id: BlockId) -> List[Felt]:
     except StarknetDevnetException as ex:
         raise RpcError(code=-1, message=ex.message) from ex
     except StarkException as ex:
+        if ex.code.name == "TRANSACTION_FAILED" and ex.code.value == 39:
```

Comment:
how can i define the code number? because i checked this is different from the cairo-lang, for instance `Invalid calldata` error does not exist in the [cairo-lang repository](src/starkware/starknet/definitions/error_codes.py)

I mean {"code": 22, "message": "Transaction Failed"}

Reply:
Your link is wrong I think. But why do you find necessary the addition of this `if` to the `except` clause?